File: PHP/is_palindrome.php
<?php
function isPalindrome($str){
  if (strrev($str) == $str){   
      return true;//palindrome
      }else{ 
      return false;//not palindrome
      }
}
//check #1
if(Palindrome("space")){//false
  echo "Palindrome";   
}else{
  echo "Not a Palindrome";   
}
//check #2
if(Palindrome("pop")){//true
  echo "Palindrome";   
}else{
  echo "Not a Palindrome";   
}
?> 

File: Go/is_palindrome.go
package main

import (
	"fmt"
	"os"
	"github.com/tpwho/isPalindrome"
)

func main() {
	mysentence := "bob"
	result := isPalindrome.Check(mysentence)
	if result == true {
		fmt.Println("The sentence is a palindrome")
		os.Exit(0)
	}
	fmt.Println("The sentence is not a palindrome")
}
